ts/window.ts

```diff
@@ -0,0 +1,77 @@
+declare global {
+  interface Window {
+    seedNodeList: any;
+
+    WebAPI: any;
+    LokiSnodeAPI: any;
+    SenderKeyAPI: any;
+    LokiMessageAPI: any;
+    StubMessageAPI: any;
+    StubAppDotNetApi: any;
+    LokiPublicChatAPI: any;
+    LokiAppDotNetServerAPI: any;
+    LokiFileServerAPI: any;
+    LokiRssAPI: any;
+  }
+}
+
+// window.WebAPI = initializeWebAPI();
+// const LokiSnodeAPI = require('./js/modules/loki_snode_api');
+// window.SenderKeyAPI = require('./js/modules/loki_sender_key_api');
+// window.lokiSnodeAPI
+// window.LokiMessageAPI = require('./js/modules/loki_message_api');
+// window.StubMessageAPI = require('./integration_test/stubs/stub_message_api');
+// window.StubAppDotNetApi = require('./integration_test/stubs/stub_app_dot_net_api');
+// window.LokiPublicChatAPI = require('./js/modules/loki_public_chat_api');
+// window.LokiAppDotNetServerAPI = require('./js/modules/loki_app_dot_net_api');
+// window.LokiFileServerAPI = require('./js/modules/loki_file_server_api');
+// window.LokiRssAPI = require('./js/modules/loki_rss_api');
+
+export const exporttts = {
+  // APIs
+  WebAPI: window.WebAPI,
+
+  // Utilities
+  Events: () => window.Events,
+  Signal: () => window.Signal,
+  Whisper: () => window.Whisper,
+  ConversationController: () => window.ConversationController,
+  passwordUtil: () => window.passwordUtil,
+
+  // Values
+  CONSTANTS: () => window.CONSTANTS,
+  versionInfo: () => window.versionInfo,
+  mnemonic: () => window.mnemonic,
+  lokiFeatureFlags: () => window.lokiFeatureFlags,
+
+  // Getters
+  getAccountManager: () => window.getAccountManager,
+  getConversations: () => window.getConversations,
+  getFriendsFromContacts: () => window.getFriendsFromContacts,
+  getSettingValue: () => window.getSettingValue,
+
+  // Setters
+  setPassword: () => window.setPassword,
```

Comment:
uhm isn't this incorrect,

with this method we would have to do the following
```ts
import { setPassword } from 'window'
setPassword()('newPassword')
```

All of the field should be:
```
<value>: window.<value>
```
e.g `setPassword: window.setPassword`
This will work because `window.setPassword` type is `(string) => void`

Reply:
Just placeholders for now